src/service/iot_config.proto

```diff
@@ -368,24 +368,24 @@ message route_stream_res_v1 {
   bytes signature = 7;
 }
 
-message session_key_filter_v1 {
-  uint64 oui = 1;
+message skf_v1 {
+  string route_id = 1;
   uint32 devaddr = 2;
   // the hex-encoded string of the binary session key
   string session_key = 3;
 }
 
-message session_key_filter_list_req_v1 {
-  uint64 oui = 1;
+message route_skf_list_req_v1 {
```

Comment:
lists all skfs attached to a single route, correct?

Reply:
Yes